transaction/src/transaction/builder.rs

```diff
@@ -271,6 +284,9 @@ impl Builder {
         self.outputs.shuffle(rng);
         self.delegations.shuffle(rng);
         self.undelegations.shuffle(rng);
+        self.validator_definitions.shuffle(rng);
+
+        println!("Shuffled... {:#?}", self.validator_definitions);
```

Comment:
nit: this should probably be a `tracing` invocation? (no worries if it's just for debugging in a WIP state, though)

Reply:
Yeah the `println` statements are just for debugging, will remove in a polish up commit